lookout/style/format/analyzer.py

```diff
@@ -172,14 +204,21 @@ def _files_by_language(cls, files: Iterable[File]) -> Dict[str, Dict[str, File]]
         return result
 
     @classmethod
-    def _load_config(cls, config: Mapping[str, Any]):
-        def recursive_update(mapping, other):
-            for key, value in other.items():
-                if isinstance(value, dict):
-                    recursive_update(mapping.setdefault(key, {}), value)
-                else:
-                    mapping[key] = value
+    def _load_analyze_config(cls, config: Mapping[str, Any]) -> Mapping[str, Any]:
+        """
+        Merges config for analyze call with default config values stored inside this function.
+
+        :param config: User-defined config.
+        :return: Full config.
+        """
+        final_config = {
+            "debug": False,
+        }
+        FormatAnalyzer.recursive_update(final_config, config)
```

Comment:
~Wow this name sucks so much!~

The best here would be to move this `recursive_update` out of the class and name it `merge_dicts`

Reply:
https://github.com/src-d/style-analyzer/pull/200